tests/framework/s3fetcher.py

```diff
@@ -143,48 +143,6 @@ def init_vm_resources(self, microvm_image_name, microvm):
                 microvm.ssh_config["ssh_key_path"] = microvm_dest_path
                 os.chmod(microvm_dest_path, 0o400)
 
-    def hardlink_vm_resources(self, microvm_image_name, from_microvm, to_microvm):
```

Comment:
Is this related to the change of this commit?

Reply:
It is not used by any test anymore.